packages/amplify-migration-codegen-e2e/src/__tests__/codegen_e2e.test.ts

```diff
@@ -0,0 +1,23 @@
+import path from 'node:path';
```

Comment:
Can we call this file `migration_codegen_e2e.test.ts`.

We have tests related other "codegen" in the repo.